semantic_conventions/metrics/thread.yaml

```
@@ -0,0 +1,10 @@
+groups:
+  - id: metric_attributes.thread
```

Are "thread" attributes only applicable to metrics? 

```suggestion
  - id: thread
```

Or (to be consistent with https://github.com/open-telemetry/semantic-conventions/blob/main/semantic_conventions/http-common.yaml#L2

```suggestion
  - id: attributes.thread.common
```

---

we have thread name/id here: https://github.com/open-telemetry/semantic-conventions/blob/9e3e528cccb3a3c0c6b4d5a057b4256e398fd69f/semantic_conventions/trace/general.yaml#L202

does it make sense to move all 3 into `thread-common.yaml`?

---

For the sake of the scope of this, I would keep as you have (decide what to call the new file/`id). Once this merges I will open an issue to discuss if the others in `trace/general.yaml` should be moved there. What do you think?

---

sounds good :smile: